### DISCUSSION TITLE
Consider renaming "Seasonal backgrounds" option

### DISCUSSION THREAD

**Discussion by djenson2000** [ppy/osu#28664] [Ideas] [answered]
I personally do not enjoy having often near NSFW characters for a main menu of a game. While this can be mitigated with the seasonal background option, it was not clear to me that that was referring to these main menu background, and only found out with discussion [#24892](https://github.com/ppy/osu/discussions/24892)

I have no issue with the [official mascots](https://osu.ppy.sh/wiki/en/Mascots) but some of these background images are really pushing it. I believe it also may put quite a few off if the first time they open the game that's what they're shown. 

To me, a seasonal background is a Christmas, Halloween etc themed background for a limited time. Though I get the backgrounds do charge with the literal seasons, I didn't find It obvious that during no known national event or holiday that that was the option causing the near NSFW backgrounds. (i.e the fact that there shown year round confused me)

**My Ideas**

1. Rename "Seasonal Backgrounds" to "Use User submitted fanart as background" or similar

   I think seasonal can be omitted as it could potentially add confusion that it would only appears at certain times.  

2. Add an option on the main menu to the option to change art sources - possibly like how microsoft does "like the image you see"

3. Have the default as art only comes from beatmaps - however this will lead to less seeing the often really nicely drawn art. 

4. Categorize the art into PG and 16+ category's, with an option in the menu to enable to PG16 art - I however believe this would be quite difficult to do due to differing opinions.

**Comment by Doksha** [comment]
or have it be "disable background art" and have another option to expand this to beatmap (and there for song selection) backgrounds. (instead of just blurring)

**Reply by djenson2000** [reply]
That's a good idea, would need some rejigging of the 2 separate dropdowns but easily possible

**Reply by Stoppedpuma** [reply]
See #13333

**Reply by djenson2000** [reply]
Ppy said a "An explicit content toggle" is coming - How are they meant to decide what is explicit. Technically none of the backgrounds are explicit, just highly suggestive. I suggested a similar option (option 4) but who is meant to tag these and on what basis.

**Comment by peppy** [comment, marked as answer]
We filter out any over-the-top suggestive backgrounds already (although I can agree this is subjective). Nothing should be "16+" in the backgrounds we ship on the main menu.

I don't believe we need to rename or change the option. It *is* seasonal because they change every season (4 times a year, sometimes with an extra one or two special events).

**Reply by djenson2000** [reply]
Though as you say it is objective, as a laptop user who plays on outdoor and public environments, I had some of my friends asking what I was playing with concerned looks. 

I agree it is literally seasonal, I said that in my post, but it's not what I expected the option to be called. Maybe that is a cultural difference, either way, fair enough. 

Would you consider something like Option 3? It could even link back to the artist if the user liked there art! Not sure if I'd be able to implement, my c# is very rusty, though if you like the idea I could look into it :) 

Thanks for your consideration
